cypress/fixtures/objects/moduleReqId/moduleReqId.json

```diff
@@ -1227,7 +1067,7 @@
                 "req": [
                     {
                         "method": {
-                            "id": "UserGrants 1.56",
+                            "id": "UserGrants 1.39",
```

Comment:
the requirement id must be properly mapped from the spec document, this looks to be just incrementing the numbers ?